src/wireviz/wv_cli.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+from pathlib import Path
+
+import click
+
+if __name__ == "__main__":
+    sys.path.insert(0, os.path.join(os.path.dirname(__file__), ".."))
+
+import wireviz.wireviz as wv
+from wireviz import APP_NAME, __version__
+from wireviz.wv_helper import open_file_read
+
+format_codes = {
+    "c": "csv",
+    "g": "gv",
+    "h": "html",
+    "p": "png",
+    "P": "pdf",
+    "s": "svg",
+    "t": "tsv",
+}
+
+epilog = "The -f or --format option accepts a string containing one or more of the "
+epilog += "following characters to specify which file types to output:\n"
+epilog += ", ".join([f"{key} ({value.upper()})" for key, value in format_codes.items()])
+
+
+@click.command(epilog=epilog, no_args_is_help=True)
```

Comment:
Is there a good reason for removing the `-h` option for help?

If not, then I suggest adding it in addition to `--help`. Then we have both help options unchanged from our previous release, and it's consistent with our other new options that all have both short and long options.
```suggestion
@click.command(
    epilog=epilog,
    no_args_is_help=True,
    context_settings=dict(help_option_names=["-h", "--help"]),
)
```